src/renderer/services/neo.js

```diff
@@ -326,17 +327,33 @@ export default {
       try {
         const inMemory = _.get(store.state.transactionDetails, hash);
         if (inMemory) {
-          inMemory.currentBlockHeight = network.getSelectedNetwork().bestBlock.index;
-          inMemory.confirmations = inMemory.currentBlockHeight - inMemory.block;
+          if (network.getSelectedNetwork().bestBlock) {
+            inMemory.currentBlockHeight = network.getSelectedNetwork().bestBlock.index;
+            inMemory.confirmations = inMemory.currentBlockHeight - inMemory.block;
+          }
           return resolve(inMemory);
         }
 
         return rpcClient.getRawTransaction(hash, 1)
           .then((transaction) => {
-            transaction.currentBlockHeight = network.getSelectedNetwork().bestBlock.index;
+            const inputPromises = [];
```

Comment:
Is there a better name for this? Maybe transactionBlock?

Reply:
Since it is now being used for the promise from getting the block header as well as the promises for getting input transactions, I could rename it to something more generic than inputPromises; like maybe transactionPromises. I'll do that.